### PR TITLE
Implement the Interactive Window interface

### PR DESCRIPTION
**Pull request by gjsjohnmurray**
Also contribute to the New File... quickpick

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,27 @@ REST Book is a Visual Studio Code extension that allows you to perform REST call
...
+code-insiders . --enable-proposed-api=tanhakabir.rest-book
+```
+
+or use the `Preferences: Configure Runtime Arguments` to add this extension's ID to the `enable-proposed-api` array, then relaunch:
+```json
+{
+    ...
+    "enable-proposed-api": ["tanhakabir.rest-book"]
+}
+```
```
Comment (gjsjohnmurray) [inline README.md:33]: @tanhakabir the extra information I added above seems to be redundant. I just verified that I was able to install 5.0.1 from Marketplace into 1.64 and us it without doing anything special to allow the proposed API. Maybe you've been able to publish the extension despite its "enabledApiProposals" clause in package.json. In my extension deleopment experience that's always barred publication.

Comment (tanhakabir) [reply README.md:33]: Thank you! Sorry I had missed this!

**Comment by gjsjohnmurray** [comment]
@tanhakabir any chance you'll be willing to merge this and publish a new version of your excellent extension? I think it'll then also provide a handy example of the Interactive Window features which are growing inside VS Code.

**Comment by tanhakabir** [comment]
Sorry @gjsjohnmurray didn't have any time during the week but trying this out now!

**Comment by tanhakabir** [comment]
Looks great thank you! Merging in

**Pull request merged**